tests/test_integration.py

```diff
@@ -56,7 +56,7 @@ def _clear_css(self, content):
 
         Perhaps I will eat these words
         """
-        clean_content = re.sub("r'<style>.*</style>", content, "", flags=re.DOTALL)
+        clean_content = re.sub("<style>.*</style>", "", content, flags=re.DOTALL)
```

Comment:
Ok I see how I had the repl and the content switched here (embarrassing)

But why drop the raw string for a regular one